tests/test_misc_table.py

```diff
@@ -54,7 +54,6 @@ def create_misc_table(
     for name in obj:
         table[name] = audformat.Column()
         table[name].set(obj[name].values)
-    table._df = table.df.astype(obj.dtypes)
```

Comment:
When I copied this function include this line to `test_database.py` and used it in `test_update()` I got an error since calling `pd.concat()` inside `MiscTable.update()` was changing the `dtype` for `int64` to `Int64`. First I thought I found a bug, but then I realized that a `MiscTable` cannot have a column with `dtype` other than `object` unless the column is assigned to a scheme. So with this line we would create invalid `Table` objects. After removing the line the tests worked, so I decided to also remove this line here and in the function `create_table()` in `test_table.py`. I then changed the `dtype` of all empty `pd.Series' to `'object', it's not really necessary to do that, but I think it makes more sense since it is anyway set to `'object'` when creating the `MiscTable` or `Table`. I had to slightly adjust one test and remove one, will put a comment there.